brokers/unified/broker_test.go

```diff
@@ -1039,7 +1039,7 @@ func TestBroker_getPluginMetas(t *testing.T) {
 				errRegexp: nil,
 				fetchURL: fmt.Sprintf(
 					RegistryURLFormat,
-					pluginFQNWithRegistry.Registry,
+					pluginFQNWithRegistry.Registry+"/plugins",
```

Comment:
It looks unformatted. Please fix it manually or execute `gofmt`

Reply:
gofmt fixes nothing for me in this place